src/NLog.Extensions.Logging/Targets/ExtensionILoggerTarget.cs

```diff
@@ -0,0 +1,173 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using NLog.Layouts;
+using NLog.Targets;
+
+namespace NLog.Extensions.Logging
+{
+    /// <summary>
+    /// Forwards NLog LogEvents to Microsoft ILogger-interface with support for NLog Layout-features
+    /// </summary>
+    [Target("ExtensionILogger")]
+    public class ExtensionILoggerTarget : TargetWithContext
+    {
+        private readonly Microsoft.Extensions.Logging.ILogger _logger;
+
+        /// <summary>
+        /// EventId forwarded to ILogger
+        /// </summary>
+        public Layout EventId { get; set; }
+
+        /// <summary>
+        /// EventId-Name forwarded to ILogger
+        /// </summary>
+        public Layout EventName { get; set; }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ExtensionILoggerTarget" /> class.
+        /// </summary>
+        /// <param name="logger">Microsoft ILogger instance</param>
+        public ExtensionILoggerTarget(Microsoft.Extensions.Logging.ILogger logger)
+        {
+            _logger = logger;
+            Layout = "${longdate}|${level:uppercase=true}|${logger}|${message:withException=true:exceptionSeparator=|}";
```

Comment:
I think the default layout should be the same for all targets. (yes, it needs a change in NLog 5)

Reply:
Change the Layout to `{message}`

Reply:
That's not the default? ;)